packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -13,6 +13,7 @@ export interface IExampleCardProps {
   dos?: JSX.Element;
   donts?: JSX.Element;
   isScrollable?: boolean;
+  codepen?: JSX.Element;
```

Comment:
Please jsdoc everything in the props...

Reply:
does that include all the props that I didn't modify? eg dos donts etc

Reply:
might as well :) short bit of work for large gain.

Reply:
Yes please. :) It's a little ironic the documentation project is lacking in documentation itself :) Everyone being willing to go the little bit of extra mile is the only way things will systematically improve (instead of continuously worsening over time).

Reply:
done :)